components/Workspaces/InsightUpgradeModal.tsx

```diff
@@ -26,19 +26,30 @@ export default function InsightUpgradeModal({
           <div className="min-w-[712px] flex flex-col gap-8">
             <DialogCloseButton onClick={onClose} />
             <section className="flex flex-col gap-2">
-              <DialogTitle className="text-xl">This Insight page is over the free Workspace limit</DialogTitle>
-              <p className="text-sm text-slate-500">
-                Your Insight page has{" "}
-                <span className="font-bold">
-                  {overLimit} {variant}
-                </span>{" "}
-                but the free Workspace only allows for{" "}
-                <span className="font-bold">
-                  {variant === "repositories" ? 100 : 10} {variant}
-                </span>{" "}
-                tracked. Don&apos;t worry, your insights won&apos;t be deleted, but if you want to continue using
-                OpenSauced you should upgrade your Workspace to a PRO Workspace.
-              </p>
+              <DialogTitle className="text-xl">
```

Comment:
We'll need to rework this to support the newer dialog/drawer component @isabensusan, but not a blocker atm.